src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -75,10 +80,13 @@ const FinancePage = () => {
           </ListItemIcon>
           Create Reimbursement Request
         </MenuItem>
-        <MenuItem onClick={() => {}}>
+        <MenuItem onClick={handleAccountCreditOpen}>
           <ListItemIcon>
             <AttachMoneyIcon fontSize="small" />
           </ListItemIcon>
+          {AccountCreditModalShow && (
```

Comment:
you don't need accountcreditModalshow &&, the modalShow prop already hides/shows the modal